src/foam/u2/dialog/NotificationMessage.js

```diff
@@ -20,111 +20,204 @@ foam.CLASS({
+      border-radius: 3px;
+      box-shadow: 0 1px 3px 0 rgba(0, 0, 0, 0.08);
+      background: #e2f2dd;
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+    }
+    ^banner {
+      background-color: #32bf5e;
+      height: 4px;
+      width: inherit;
+      max-width: 1072px;
+      margin-left: -24px;
+      border-top-left-radius: 3px;
+      border-top-right-radius: 3px;
+      position: absolute;
+      top: 0;
     }
     @keyframes fade {
       0% { opacity: 0; }
       10% { opacity: 1; }
       80% { opacity: 1; }
       100% { opacity: 0; }
     }
-    ^error-background{
-      background: #f4cccc;
-      border: 1px solid #f33d3d;
+    ^status-icon {
+      width: 20px;
+      height: 20px;
+      margin-right: 16px;
+      vertical-align: middle;
     }
-    ^ .close-x{
-      right: 5px;
-      top: 10px;
+    ^content {
+      display: inline-block;
+      vertical-align: middle;
+      font-family: IBMPlexSans;
+      font-size: 16px;
+      font-weight: 600;
+      color: #19402e;
+      letter-spacing: normal;
     }
-    ^ .foam-u2-ActionView-close{
-      width: 30px;
-      height: 30px;
-      position: absolute;
-      left: 0px;
-      top: -5px;
-      z-index: 101;
-      opacity: 0.01;
+    ^description {
+      font-family: IBMPlexSans;
+      font-size: 14px;
+      font-weight: normal;
+      line-height: 1.43;
+      letter-spacing: normal;
+      color: #19402e;
+      margin-left: 36px;
     }
-    ^ .close-x {
-      position: absolute;
-      width: 32px;
-      height: 32px;
-      opacity: 0.3;
+    ^error-background {
+      background: #fff6f6;
     }
-    ^ .close-x:hover {
-      opacity: 1;
+    ^warning-background {
+      background: #fdf8de;
     }
-    ^ .close-x:before, .close-x:after {
-      position: absolute;
-      content: ' ';
-      height: 20px;
-      width: 2px;
-      background-color: #333;
+    ^info-background {
+      background: #e5f1fc;
+    }
+    ^error-banner {
+      background: #d9170e;
+    }
+    ^warning-banner {
+      background: #eedc00;
     }
-    ^ .close-x:before {
-      transform: rotate(45deg);
+    ^info-banner {
+      background: #406dea;
     }
-    ^ .close-x:after {
-      transform: rotate(-45deg);
+    ^error-content {
+      color: #631414;
+    }
+    ^warning-content {
+      color: #816819;
+    }
+    ^info-content {
+      color: #202341;
+    }
+    ^link-icon {
+      display: inline-block;
+      margin-top: 2px;
+      vertical-align: middle;
+      margin-right: 0 !important;
+      width: 16px;
+      height: 16px;
+    }
+    ^close-icon {
+      background-image: url("images/round-close-icon.svg");
+      background-size: 12px 12px;
+      cursor:pointer;
+      height: 12px;
+      opacity: 0.5;
+      width: 12px;
+      position: absolute;
+      top: 18;
+    }
+    ^close-icon:hover {
+      opacity: 1;
     }
   `,
 
+  properties: [
+    {
+      class: 'String',
+      name: 'type'
```

Comment:
Can this use the same enum from foam.log.LogLevel?

Reply:
Its similar but I don't think so due to the fact that we need a success state, also the coloring isn't the same unless we plan to unify them. I'm up for the discussion though.

Reply:
re: colours - I don't really care what shade of green log info messages are. 
re: success - for notification? I don't see this anywhere - and  clearly shows the issue with free form text for state.  How would anyone to know what the 'type's are supposed to be.  I see notifications states identical to log messages: info, warning, error.  Where/how is success used?   @nanoscott tagging you here to comment as well if possible. Thanks.

Reply:
info and success are basically the same .. scenarios where this is something we would want to display:
- Record was successfully created
- Approval request was approved
- etc.
All of these things are fine being shown as info messages. So using LogLevel should be fine.

Reply:
So we are going to do the LogLevel migration in a follow-up task, not in this PR. We can resolve this comment in this PR and address it later.